doc/odoc.mld

```diff
@@ -1,19 +1,19 @@
-{0 odoc}
+{0 [odoc]}
 
 For a quick look at the [odoc] syntax, see the {{!cheatsheet}cheatsheet}!
 
 {1:overview What is [odoc]?}
 
 [odoc] is a documentation generator for OCaml. It reads doc comments
-from your source files and your [.mld] files, and outputs HTML, LaTeX and
+from your source files and your [.mld] files, then outputs HTML, LaTeX and
 man pages. The pages you are reading now are rendered using [odoc].
 
 Text inside doc comments (delimited by [(** ... *)]) is marked up in
 [odoc] syntax:
 
 {[
 val float_dsig : int -> float t
-(** [float_dsig d] rounds the normalized {e decimal} significand
+(** [float_dsig d] rounds the normalised {e decimal} significant
```

Comment:
I think "significand" is [intended](https://en.wikipedia.org/wiki/Significand).

Also, this part is actually a quote: https://ocaml.org/p/fmt/0.8.3/doc/Fmt/index.html#val-float_dsig
So I don't think we should modify it?

Reply:
Oh good to know! Thank you!